scripts/create_release_pr.sh

```diff
@@ -11,7 +11,7 @@ fi
 
 # Create PR title
 today=$(date +%Y-%m-%d)
-release_count=$(gh pr list --state all --search "Release $today in:title" | grep "Release $today" |wc -l)
+release_count=$(gh pr list --state all --search "Release $today in:title" | grep -c "Release $today" || true)
```

Comment:
There were two problems:

1. Arithmetic error: The `wc -l` command returned a value with leading spaces, which caused bash arithmetic to fail when trying to calculate `release_number=$((release_count + 1))`.
2. Pipeline failure: The grep command would fail when no matching PRs were found, causing the script to exit due to `set -e`.

Both issues were fixed by:
- Using `grep -c` instead of `grep | wc -l` to count matches more reliably
- Adding `|| true` to prevent the script from exiting when `grep` finds no matches